src/main/java/com/ldtteam/structurize/util/BlockUtils.java

```diff
@@ -395,6 +410,18 @@ public static void removeFluid(World world, BlockPos pos)
         }
     }
 
+    /**
+     * A simple check to fetch the default fluid block for this dimension
+     * @param world the world of the dimension
+     * @return the default blockstate for the default fluid
+     */
+    public static BlockState getFluidForDimension(World world)
+    {
+        return world.getDimensionKey().equals(World.THE_NETHER)
```

Comment:
This is only for The nether and not for netherlike dimensions though. Since it's not taking the DImenstionType